README.md

```diff
@@ -44,20 +46,30 @@ going to assume programmatic access to a Kubeflow installation. Such a deploymen
 very least) of the following pieces:
 
 * A **Kubernetes cluster**, e.g.
+    * Canonical Kubernetes
     * MicroK8s
-    * Charmed Kubernetes
     * EKS cluster
-    * AKS cluster <!-- codespell-ignore -->
+    * AKS cluster
 * **Charmed Kubeflow** deployed on top of it
 * **MLflow (optional)** deployed alongside Kubeflow
 
+To run the tests on [Canonical Kubernetes](https://documentation.ubuntu.com/canonical-kubernetes/release-1.32/), we provide a `concierge.yaml` file in this repository for conveniently deploying a cluster with [concierge](https://github.com/canonical/concierge).
+
```

Comment:
I see only this diff in the README.md is relevant to the concierge change, is it intentional to include the other changes?